File: JS/Conditional Statements/LunchBreak.js
function calcLunchBreak(input){
    let movie = input[0];
    let movieTime = Number(input[1]);
    let breakTime = Number(input[2]);
    let lunchTime = breakTime / 8;
    let restTime = breakTime / 4;
    let timeLeft = breakTime - lunchTime - restTime;
    let difference = Math.ceil(Math.abs(timeLeft - movieTime));
    if (timeLeft >= movieTime){
        console.log(`You have enough time to watch ${movie} and left with ${difference} minutes free time.`);
    }
    else{
        console.log(`You don't have enough time to watch ${movie}, you need ${difference} more minutes.`);
    }
}

// calcLunchBreak(["Game of Thrones", "60", "96"])

File: Python Basics/exercises/lunch_break.py
import math

movie = input()
movie_lenght = int(input())
break_time = int(input())

lunch_time = break_time / 8
rest_time = break_time / 4
time_left = break_time - lunch_time - rest_time
time_left_after_movie = abs(time_left - movie_lenght)
time_left_after_movie = math.ceil(time_left_after_movie)

if time_left >= movie_lenght:
    print(f"You have enough time to watch {movie} and left with {time_left_after_movie} minutes free time.")
else:
    print(f"You don't have enough time to watch {movie}, you need {time_left_after_movie} more minutes.")